plugins/command-manager/src/main/java/com/wazuh/commandmanager/model/Orders.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2024, Wazuh Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.wazuh.commandmanager.model;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.search.SearchHit;
+import org.opensearch.search.SearchHits;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/** Helper class for managing command orders */
+public class Orders {
+
+    private static final Logger log = LogManager.getLogger(Orders.class);
+
+    /**
+     * Helper static method that takes the search results in SearchHits form and parses them into
+     * Order objects. It then puts together a json string meant for sending over HTTP
+     *
+     * @param searchHits the commands search result
+     * @return A json string payload with an array of orders to be processed
+     */
+    public static String getOrders(SearchHits searchHits) {
```

Comment:
This class should implement the `ToXContent` interface as the rest of models.

Ideally, the model class should have:

- A public constructor.
- A private array of Orders.
- A `static public fromSearchHits(SearchHits hits)` method, as a builder method. It will parse the search hits and add the orders to the `orders` array. 
- An implementation of the `toXContent()` method.

Reply:
Solved in the last commit. The sending of commands looks as follows:

```json
14:17:35 DEBUG i.g.i.h.AbstractResourceMatcher - Matched resource config for POST https://127.0.0.1:55000/orders
{
  "orders": [
    {
      "source": "Engine",
      "user": "user53",
      "target": {
        "type": "agent",
        "id": "target4"
      },
      "action": {
        "name": "restart",
        "args": [
          "/path/to/executable/arg6",
          "/path/to/executable/arg6",
          "/path/to/executable/arg6",
          "/path/to/executable/arg6"
        ],
        "version": "v4"
      },
      "document_id": "s6825JMBY5heyVu78wQP"
    }
  ]
}
```